build.sbt

```diff
@@ -242,11 +239,16 @@ def multiJvmTestSettings: Seq[Setting[_]] = {
       executeTests in Test := {
         val testResults = (executeTests in Test).value
         val multiNodeResults = (executeTests in MultiJvm).value
-        val overall =
-          if (testResults.overall.id < multiNodeResults.overall.id)
-            multiNodeResults.overall
-          else
-            testResults.overall
+        import TestResult.{ Passed, Failed, Error }
+        val overall = (testResults.overall, multiNodeResults.overall) match {
+          case (Passed, Passed)                    => Passed
+          case (Failed, Failed)                    => Failed
+          case (Error, Error)                      => Error
+          case (Passed, Failed) | (Failed, Passed) => Failed
+          case (Passed, Error) | (Error, Passed)   => Error
+          case (Failed, Error) | (Error, Failed)   => Error
```

Comment:
Is `case (a,b)` possible? (I guess if someone added another `value` on the `TestResult` `Enumeration`).

Reply:
`TestResult` isn't an enumeration any more.  But I _thought_ it was unsealed, but that's not true: https://github.com/sbt/sbt/blob/v1.2.8/testing/src/main/contraband-scala/sbt/protocol/testing/TestResult.scala

I'll drop `case (a, b)`.